### ISSUE TITLE
JAWS reads last list item along with first list item for controls inside "Colors" toolbar of CKEditor

### ISSUE THREAD

**Issue by sagar17sep** [ckeditor/ckeditor-dev#1947]
BUG

## Provide detailed reproduction steps (if any)

1. Bring Focus on CK Editor toolbar and Navigate to CK Editor toolbar using Alt + F10 and navigate to controls inside "Colors" toolbar.
2. Start JAWS.

### Expected result

JAWS should read the name of the controls and all its list items correctly and no two list items should be read together.

### Actual result

JAWS reads last list item(More Colors...) along with first list item(Automatic) for controls - "Text Color and Background color" inside "Colors" toolbar of CK Editor.

## Other details

* Browser: IE 11.309.16299.0
* OS: Windows 10
* CKEditor version: 4.6.2
* Installed CKEditor plugins: …

**Comment by mlewand** [comment]
I'm sorry but I don't understand the TC. Are you activating color dropdown, and _then starting JAWS_? In which case the dropdown will be dismissed. So nothing like that should happen.

Please provide step by step reproduction case.

Also, please note that:
* we do not support IE11 with JAWS, please use Firefox in ESR / latest version
* you should use the latest available CKEditor version

**Comment by sagar17sep** [comment]
Please find the steps in more detail-

1.) Start JAWS.
2.) Bring Focus on CK Editor toolbar and Navigate to CK Editor toolbar using Alt + F10 and navigate to controls inside "Colors" toolbar.
3.) When the focus is on Text color/Background color color toolbar in CK editor and when we press enter and bring focus on Automatic, It reads "automatic" and "more color" label together which should not be done. It should not combine the labels together.

**Comment by sagar17sep** [comment]
Could you please update on above mentioned issue.

**Comment by mlewand** [comment]
@sagar17sep Sorry I was testing it using Firefox - where the problem does not occur. After checking it with IE8/IE11 I can confirm that the issue described by you occurs.

However, please refer to our [Supported Platform section](https://docs.ckeditor.com/ckeditor4/latest/guide/dev_a11y.html#supported-platforms) in Accessibiltiy Support guide. You'll see that our platform used for testing CKEditor is Firefox and JAWS.

It does look like an upstream issue between IE and JAWS. I can't tell really exactly which app is causing the problem - but it does not occurs with our supported screen reader setup.
